lang/python/python3-pyparsing/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2019 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pyparsing
+PKG_VERSION:=2.4.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pyparsing/
+PKG_HASH:=1873c03321fc118f4e9746baf201ff990ceb915f433f23b395f5580d1840cb2a
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pyparsing
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Library for constructing grammar directly in python
+  URL:=https://github.com/pyparsing/pyparsing/
+  DEPENDS:=+python3-light
+  VARIANT:=python3
+endef
+
+define Package/python3-pyparsing/description
+  The pyparsing module is an alternative approach to creating and executing simple grammars,
```

Comment:
Nitpick: in these two rows you have long indent and in the other two, you don't have it. Stick with two spaces as you have in the last two rows.

Reply:
fixed to two spaces